docs/sources/operator/architecture.md

```diff
@@ -115,11 +114,11 @@ anywhere between 1/N to N targets to reshuffle.
 The sharding mechanism is borrowed from the Prometheus Operator.
 
 The number of replicas can be defined, similarly to the number of shards. This
-creates duplicate shards. This must be paired with a `remote_write` system that
-can perform HA duplication. Grafana Cloud and Cortex provide this out of the
+creates deduplicate shards. This must be paired with a `remote_write` system that
```

Comment:
```suggestion
creates duplicate shards. This must be paired with a `remote_write` system that
```
In this case it's the duplicated shards that require de-duplication by the system that is ingesting metrics/logs.